AppDB/backup/cassandra_backup.py

```diff
@@ -0,0 +1,352 @@
+""" Cassandra data backup. """
+
+import logging
+import os
+import statvfs
+import sys
+import tarfile
+import subprocess
+from subprocess import call
+from subprocess import CalledProcessError
+from os.path import getsize
+
+import backup_exceptions
+import gcs_helper
+import backup_recovery_helper
+from backup_recovery_constants import BACKUP_DIR_LOCATION
+from backup_recovery_constants import BACKUP_FILE_LOCATION
+from backup_recovery_constants import BACKUP_ROLLBACK_SUFFIX
+from backup_recovery_constants import CASSANDRA_DATA_SUBDIRS
+from backup_recovery_constants import PADDING_PERCENTAGE
+from backup_recovery_constants import StorageTypes
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../"))
+import dbconstants
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../cassandra/"))
+import start_cassandra
```

Comment:
Would it be cleaner to do something like `from cassandra import start_cassandra`, etc. to reduce the number of times the sys path is modified?